rece1.py

```diff
@@ -39,7 +39,7 @@
 DEBUG = False
 XML = False
 TIMEOUT = 30
-CONFIG_FILE = "rece.ini"
+CONFIG_FILE = "conf/rece.ini"
```

Comment:
- [ ] Please change back this to rece.ini, you can copy the file in the GitHub Actions

Reply:
made the required changes